src/meta/src/barrier/context/mod.rs

```diff
@@ -99,6 +100,11 @@ pub(super) trait GlobalBarrierWorkerContext: Send + Sync + 'static {
         &self,
         refresh_finished_table_job_ids: Vec<JobId>,
     ) -> impl Future<Output = MetaResult<()>> + Send + '_;
+
```

Comment:
I think we can call it in `reload_xxx_info` before we mark the dirty epochs as aborted instead of introducing a new trait method.